README.md

```diff
@@ -289,6 +289,35 @@ final class Foo
     }
 }
 ```
+
+### Codor.Classes.NewInstance ###
+Classes should not contain new instance. Should use depency injection.
+
+:x:
+```php
+class NewInConstructor
+{
+    protected MyClass $myClass;
+
+    public function __construct()
+    {
+        $this->myClass = new MyClass();
+    }
+}
+```
+
+:white_check_mark:
+```php
+class NewInConstructor
+{
+    private MyClass $myClass;
```

Comment:
The visibility of the property changed from `protected` to `private` between the two samples.
I think it should the same (pick the one you prefer).